source/blueprints/banking.rst

```diff
@@ -86,253 +82,187 @@ This should create a MindMeld project folder called ``food_ordering`` in your cu
-
-.. warning::
-
-   Make sure that Elasticsearch is running in a separate shell before invoking the QuestionAnswerer.
-
+    #the app auto fill indicates it is a dialogue state which requires a form and uses slot filling logic	
+    @app.auto_fill(intent='check_balances', form=balance_form)
+    #the function will work as if the required entity has been provided 
+    def check_balances_handler(request, responder):
+        if not user_data: 
+            _pull_data(request)
+        if request.entities:
+            for entity in request.entities:
+                if entity['type'] == 'account_type':
+                    responder.slots['account'] = entity['value'][0]['cname'] or entity['text']
+                    responder.slots['amount'] = _get(entity['value'][0]['cname'] or entity['text'])
+                    responder.reply('Your {account} account balance is {amount}')
+        
 .. code:: python
 
-   from mindmeld.components.question_answerer import QuestionAnswerer
-   qa = QuestionAnswerer(app_path='food_ordering')
-   qa.get(index='menu_items')[0]
-
-.. code-block:: console
-
-    {
-       "size_group": "pizzasize",
-       "menu_id": "57572a43-f9fc-4a1c-96fe-788d544b1f2d",
-       "restaurant_id": "B01DEEGQBK",
-       "size_prices": [
-          {
-             "name": "12\" Small",
-             "price": 13.99,
-             "id": "B01N9YUFMX"
-          },
-          {
-             "name": "14\" Medium",
-             "price": 16.99,
-             "id": "B01MRDF6V1"
-          },
-          {
-             "name": "16\" Large",
-             "price": 17.99,
-             "id": "B01MUI0ZGE"
-          },
-          {
-             "name": "18\" X-Large",
-             "price": 19.99,
-             "id": "B01N7ZK0ZR"
-          }
-       ],
-       "option_groups": [
-          "pizzawhole"
-       ],
-       "img_url": null,
-       "description": null,
-       "id": "B01NB08SGM",
-       "popular": false,
-       "name": "Cheese Pizza",
-       "category": "Pizzas",
-       "price": 13.99
-    }
-
-.. admonition:: Exercise
+To learn more about slot filling please take a look at...
 
-   The blueprint comes with a pre-configured, pre-populated knowledge base to help you get up and running quickly. Read the User Guide section on :doc:`Question Answerer <../userguide/kb>` to learn how to create knowledge base indexes from scratch. Then, try creating one or more knowledge base indexes for your own data.
+6. Knowledge Base
+^^^^^^^^^^^^^^^^^
 
+Since the banking blueprint is a will require personal information we have decided to not include a knowledge base to mimic how this data may be received from an
+external api.
```

Comment:
from a secure server via a custom API. 

Maybe we should include the description of the context user_name/token and retrieval here instead of at the beginning.